src/components/dashboard/main-content/course-enrollments/course-cards/unenroll/UnenrollModal.jsx

```diff
@@ -76,6 +110,11 @@ const UnenrollModal = ({
           >
             Unenroll
           </StatefulButton>
+          <Button
+            onClick={updateQueryForUnenrollment}
+          >
+            Optimistic Update
+          </Button>
```

Comment:
Assuming this should be removed (left over from local QA)?

Reply:
Yep removed 👍🏽